data/mods/DinoMod/monsters/juvenile.json

```diff
@@ -90,25 +90,38 @@
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "CATFOOD", "GUILT", "NO_BREED", "CANPLAY" ]
   },
   {
-    "id": "mon_tyrannosaurus_juvenile",
+    "id": "mon_albertosaurus_juvenile",
     "type": "MONSTER",
-    "default_faction": "tyrannosaurus_juvenile",
-    "copy-from": "mon_albertosaurus_juvenile",
-    "color": "red_white",
+    "name": "red and white two-legged juvenile",
+    "description": "A small bipedal meat-eating dinosaur juvenile with huge shiny eyes, it could be from a number of different species.",
+    "default_faction": "albertosaurus_juvenile",
+    "copy-from": "mon_gallimimus_juvenile",
+    "color": "light_red_white",
     "dodge": 5,
-    "upgrades": { "age_grow": 365, "into": "mon_tyrannosaurus" },
+    "upgrades": { "age_grow": 365, "into": "mon_albertosaurus" },
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "CATFOOD", "GUILT", "NO_BREED", "CANPLAY" ]
   },
   {
-    "id": "mon_albertosaurus_juvenile",
+    "id": "mon_qianzhousaurus_juvenile",
     "type": "MONSTER",
     "name": "red and white two-legged juvenile",
-    "description": "A small bipedal meat-eating dinosaur juvenile with huge shiny eyes, it could be from a number of different species.",
-    "default_faction": "albertosaurus_juvenile",
     "copy-from": "mon_gallimimus_juvenile",
+    "description": "A small bipedal meat-eating dinosaur juvenile with huge shiny eyes, it could be from a number of different species.",
```

Comment:
```suggestion
    "description": "A small, bipedal, meat-eating dinosaur juvenile with huge shiny eyes, it could be from a number of different species.",
```